src/sections/staking/StakingPage.utils.ts

```diff
@@ -32,8 +39,26 @@ const CONVICTIONS: { [key: string]: number } = {
 const lengthOfStaking = BN(50400) // min. amount of block for how long we want to calculate APR from = one week
 const blocksPerYear = 2628000
 
+/* constants that might be changed */
+const a = "20000000000000000"
+const b = "2000"
```

Comment:
Is there a better name for these constants? I don't know what it means when I look at this code.

Reply:
I don't know what those constants mean, I pass them to wasm function that has the same argument names

Reply:
I got those constants from runtime team